factory/.env

```diff
@@ -0,0 +1,37 @@
+## This env represents the latest versions cyrpess supports. If you wish to push an older docker image,
+## please modify circle ci to release a one off branch instead of merging old versions to this file in master.
+
+# The debian image the factory is based on
+BASE_IMAGE='debian:bullseye-slim'
+
+# Node Versions: https://nodejs.org/en/download/releases/
+NODE_VERSION='16.18.1'
+
+# Update this to deploy the docker factory if you make changes to factory.Dockerfile or install scripts
+FACTORY_VERSION='1.0.0'
+
+# Chrome versions: https://www.ubuntuupdates.org/package/google_chrome/stable/main/base/google-chrome-stable
+CHROME_VERSION='107.0.5304.121-1'
+
+# Cypress versions: https://www.npmjs.com/package/cypress
+CYPRESS_VERSION='12.2.0'
+
+# Edge versions: https://packages.microsoft.com/repos/edge/pool/main/m/microsoft-edge-stable/
+EDGE_VERSION='100.0.1185.29-1'
+
+# Firefox versions: https://download-installer.cdn.mozilla.net/pub/firefox/releases/
+FIREFOX_VERSION='107.0'
+
+# Yarn versions: https://www.npmjs.com/package/yarn
+YARN_VERSION='1.22.19'
+
+# Webkit versions: https://www.npmjs.com/package/playwright-webkit
+# WEBKIT_VERSION='1.29.0'
+
+
+# Tags used for the docker images generated from the factory. Keep in mind the docker images will only release if these versions change.
+BASE_IMAGE_TAG="${NODE_VERSION}"
+
+BROWSERS_IMAGE_TAG="node-${NODE_VERSION}-chrome-${CHROME_VERSION}-ff-${FIREFOX_VERSION}-edge-${EDGE_VERSION}"
+
+INCLUDED_IMAGE_TAG="cypress-${CYPRESS_VERSION}-${BROWSERS_IMAGE_TAG}"
```

Comment:
This is a new style of version, previously the version was just the cypress version, however with that version style we'd only release a new version of included when the version of cypress was updated and it would not get updated when browser or node versions are updated